packages/terra-menu/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "terra-menu",
+  "main": "lib/Menu.scss",
+  "private": true,
+  "version": "0.1.0",
```

Comment:
We need to bump this to 0.2.0 to match https://github.com/cerner/terra-menu/blob/master/package.json#L3. When we release this next, we'll release the changes as `0.3.0`